File: C++/Week-3/01_count_digits.cpp
//1.Write a program to count the number of digits in a number.

#include <iostream>

using namespace std;

int main(){
	int n, count = 0;
	cin>>n;
	while(n != 0){
		++count;
		n /= 10;
	}
	cout<<count<<endl;
	return 0;
}

File: Python/Week-3/01_count_digits.py
#1.Write a program to count the number of digits in a number.

n = int(input())

count = 0

while n != 0:
	n = int(n / 10)
	count += 1

print(count)



# An easier way
'''
n = input()
count = len(n)
print(count)'''
